subsys/net/lib/nrf_cloud/src/nrf_cloud_jwt.c

```diff
@@ -35,21 +474,6 @@ int nrf_cloud_jwt_generate(uint32_t time_valid_s, char *const jwt_buf, size_t jw
 		.jwt_sz = jwt_buf_sz
 	};
 
-	/* Check if modem time is valid */
-	err = nrf_modem_at_cmd(buf, sizeof(buf), GET_TIME_CMD);
-	if (err != 0) {
-		LOG_ERR("Modem does not have valid date/time, JWT not generated");
-		return -ETIME;
-	}
```

Comment:
I agree, I don't think this is necessary

But I do want to understand what happens if `nrf_cloud_jwt_generate` is called without it. Do we just get an otherwise normal JWT that'll be rejected due to the invalid date?

Reply:
this has not been removed, only moved.